src/ucp/core/ucp_context.c

```diff
@@ -126,11 +126,21 @@ static ucs_config_field_t ucp_config_table[] = {
    "the eager_zcopy protocol",
    ucs_offsetof(ucp_config_t, ctx.rndv_perf_diff), UCS_CONFIG_TYPE_DOUBLE},
 
-  {"MAX_EAGER_LANES", "1",
+  {"MAX_EAGER_LANES", NULL,
+   "Maximal number of devices on which an eager operation may be executed in parallel "
+   "(alias to MAX_EAGER_RAILS)",
```

Comment:
The user sees the env params with the prefix **UCX_**  
(alias to UCX_MAX_EAGER_RAILS)
same for line 140 please

Reply:
no need to mwntion this at all, `ucx_info -f ` already would print that X is alias of Y

Reply:
yep, found it in sources... parser should be separated to own lib :)